template_content/smart_contracts/{% if deployment_language == 'python' %}config.py{% endif %}

```diff
@@ -42,11 +43,11 @@ def deploy(
             )
             deploy_response = app_client.deploy(
                 on_schema_break=(
-                    OnSchemaBreak.ReplaceApp if is_local else OnSchemaBreak.Fail
+                    OnSchemaBreak.AppendApp if is_main else OnSchemaBreak.ReplaceApp
                 ),
-                on_update=OnUpdate.UpdateApp if is_local else OnUpdate.Fail,
-                allow_delete=is_local,
-                allow_update=is_local,
+                on_update=OnUpdate.AppendApp if is_main else OnUpdate.UpdateApp,
+                allow_delete=is_main,
+                allow_update=is_main,
```

Comment:
I think you want `not is_main` here for `allow_delete` and `allow_update`?

Reply:
LocalNet & TestNet
- on_schema_break = OnSchemaBreak.ReplaceApp
- on_update = OnUpdate.UpdateApp

MainNet
- on_schema_break = OnSchemaBreak.AppendApp
- on_update = OnSchemaBreak.AppendApp

we want append for mainnet right?